### ISSUE TITLE
ts-lint support for codelyzer rules

### ISSUE THREAD

**Issue by siddharth1903** [codacy/codacy-meta#420]
Hi,

We're looking to additionally implement, on top of tslint, a few codelyzer rules related to accessibility of sites. Codelyzer is a widely used amongst the angular community for static code analysis.
https://github.com/mgechev/codelyzer/

Can you guys please let me know how do i enable rules codelyzer exposes on top of tslint?

Specific rules i'm looking for
  "template-accessibility-alt-text": true,
    "template-accessibility-elements-content": true,
    "template-accessibility-label-for": true,
    "template-accessibility-tabindex-no-positive": true,
    "template-accessibility-table-scope": true,
    "template-accessibility-valid-aria": true,
    "template-click-events-have-key-events": true,
    "template-mouse-events-have-key-events": true,
    "template-no-autofocus": true,
    "template-no-distracting-elements": true

**Comment by lolgab** [comment]
Hi @siddharth1903,
I gave a quick look to the codelyzer plugin and it seems that the authors moved their support to an ESLint plugin that offers the same functionality: https://github.com/angular-eslint/angular-eslint .
I think we are going to support `angular-eslint` instead.
Is that of for your use case @siddharth1903 ?

Lorenzo

**Comment by siddharth1903** [comment]
@lolgab Yes... our projects internally are using these ever since they have been released by tslint... We will be ofcourse able to adapt when eslint adapts these rules too... Until then we are bound to go with tslint.. is there a short term roadmap to enable for tslint and move subsequently to eslint.. I guess all angular projects are still scaffolding code with tslint until today!

**Comment by siddharth1903** [comment]
Guys, any update on this?
